RestApisGen/DataType.cs

```diff
@@ -19,6 +19,8 @@ public class ApiEndpoint
 
         public string ReservedName { get; set; }
```

Comment:
You may remove the `ReservedName` and merge it into the `ReservedNames`. Having both of them is somewhat confusing.